components/input/input.js

```diff
@@ -76,21 +78,21 @@ class Input extends Component {
       value,
     } = this.props;
     const mods = this.props.mods ? this.props.mods.slice() : [];
-
-    const otherProps = ejectOtherProps(this.props, Input.propTypes);
```

Comment:
What is the reason for getting rid of `otherProps`?

Reply:
Because we have such a task. We need to define what props are these 'otherProps'

Reply:
That's why I ask everyone to put more details into their PR's description. To reduce this kind of questions.